tests/cql/CqlComparisonOperatorsTest.xml

```diff
@@ -763,6 +775,18 @@
 			<expression>Tuple{ Id : 1, Name : 'John' } != Tuple{ Id : 2, Name : 'John' }</expression>
 			<output>true</output>
 		</test>
+		<test name="TupleNotEqJohn1John2WithNullName">
+			<expression>Tuple{ Id : 1, Name : 'John' } != Tuple{ Id : 2, Name : null }</expression>
+			<output>true</output>
```

Comment:
Same here, this should be `null`

Reply:
And as long as we always ensured that the `!=` was equivalent to a `!` of `=`, then that should be okay.

Reply:
Resolving as per the thread above, let me know if there was anything else needed here.